core/service/bytes-service.js

```diff
@@ -1,62 +1,37 @@
-var Montage = require("montage").Montage,
-    application = require("montage/core/application").application;
+var Montage = require("montage").Montage;
 
 //FIXME: quick workaround
 exports.BytesService = Montage.specialize({
```

Comment:
I think we might start with simple "tools" services, each dedicated to a single concern (here, making bytes sizes human readable). That way, we will avoid ending up with a huge and hard to maintain service. As business rules and data processing services will move in SectionServices and Repository respectively, it may not be useful to group "tools" services in their own entity (they can live in core/service), at least not right now.